config/install/core.entity_form_display.paragraph.card_event.default.yml

```diff
@@ -61,9 +62,14 @@ content:
     type: string_textarea
     weight: 5
     settings:
-      placeholder: ''
       rows: 5
-    third_party_settings: {  }
+      placeholder: ''
+    third_party_settings:
+      maxlength:
```

Comment:
@vincent-gao we also need a hook_update to update this config change. Please make sure that the hook_update only changes the `third_party_settings` of the field if `maxlength` hasn't been set.

Reply:
added a hook_update to `tide_landing_page.install` file.